stdlib/public/Concurrency/Task.swift

```diff
@@ -495,6 +495,27 @@ extension Task {
   public static func yield() async {
     fatalError("\(#function) not implemented yet.")
   }
+
+  /// Suspends the current task for _at least_ the given duration
+  /// in nanoseconds.
+  ///
+  /// This function does _not_ block the underlying thread.
+  public static func sleep(_ duration: UInt64) async {
+
+    // Set up the job flags for a new task.
+    var flags = JobFlags()
+    flags.kind = .task
+    flags.priority = .default
+    flags.isFuture = true
+
+    // Create the asynchronous task future.
+    let (task, _) = Builtin.createAsyncTaskFuture(flags.bits, nil, {})
```

Comment:
I wonder if it matters to pass in the parent... maybe for debugging it might be better to get the current task and tread this as if a child after all, just in case someone "stops execution" and happens to end up in the sleep somewhere... (Can this even happen...?)